go/filemanager/pfsmodules/chunk.go

```diff
@@ -61,38 +62,22 @@ func ParseChunk(path string) (*Chunk, error) {
 	return &cmd, nil
 }
 
-// LoadChunkData loads a specified chunk to io.Writer.
-func (p *Chunk) LoadChunkData(w io.Writer) error {
-	f, err := os.Open(p.Path)
-	if err != nil {
-		return err
-	}
-	defer Close(f)
-
-	_, err = f.Seek(p.Offset, 0)
-	if err != nil {
-		return err
-	}
-
-	loaded, err := io.CopyN(w, f, p.Size)
-	log.V(2).Infof("loaded:%d\n", loaded)
-	return err
+// Chunk is struct.
+type Chunk struct {
+	Offset   int64
+	Len      int64
+	Checksum string
```

Comment:
`ChunkMeta` have `Offset, Len, Checksum`, do we still need them here?
Given only `[]byte` is left if we remove them , maybe instead of defining a new type, we can just use `[]byte` to represent `Chunk`?